extern/boostd-data/svc/setup_yugabyte_test_util.go

```diff
@@ -15,37 +18,128 @@ var tlog = logging.Logger("ybtest")
 
 var TestYugabyteSettings = yugabyte.DBSettings{
 	Hosts:         []string{"yugabyte"},
-	ConnectString: "postgresql://postgres:postgres@yugabyte:5433",
+	ConnectString: "postgresql://postgres:postgres@yugabyte:5433?sslmode=disable",
 }
 
-// Use when testing against a local yugabyte instance.
-// Warning: This will delete all tables in the local yugabyte instance.
+// Used when testing against a local yugabyte instance.
 var TestYugabyteSettingsLocal = yugabyte.DBSettings{
 	Hosts:         []string{"localhost"},
-	ConnectString: "postgresql://postgres:postgres@localhost:5433",
+	ConnectString: "postgresql://postgres:postgres@localhost:5433?sslmode=disable",
 }
 
-func SetupYugabyte(t *testing.T) {
-	ctx := context.Background()
+func init() {
+	// Uncomment when testing against a local yugabyte instance.
+	//TestYugabyteSettings = TestYugabyteSettingsLocal
+}
```

Comment:
We could probably just move this to `setup_yugabyte_test_util_test.go` which should be compiled only if you run `go test`, no?

Reply:
We only want it when running `go test` on a local machine though (not in CI)